Assets/Scripts/Tile.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Tile
+{
+    private bool pathTile = false;
+    private bool taken = false;
+    private Vector3 position = new Vector3(0.0f, 0.0f, 0.0f);
+
+    // constructor accepts if its a path tile, if has a tower on top
+    //and a position on the screen
+    public Tile(bool vPathTile, bool vTaken, Vector3 vPosition)
```

Comment:
Does the constructor need whether it's taken or not? A tile will never have a tower on it when it's first created.